src/components/AttendantForm.test.jsx

```diff
@@ -0,0 +1,151 @@
+import AttendantForm from "./AttendantForm";
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import ErrorMessages from "../constants/errorMessages";
+
+const jobTitles = ["Accountant", "Engineer", "Lawyer", "Teacher"];
+
+const fillAndSubmitForm = async () => {
+  userEvent.type(screen.getByTestId("name"), "John");
+  userEvent.type(screen.getByTestId("last-name"), "Doe");
+  userEvent.selectOptions(await screen.findByTestId("job-title"), "Engineer");
+  userEvent.type(screen.getByTestId("age"), "40");
+  userEvent.click(screen.getByTestId("submit"));
+};
+
+describe("<AttendantForm />", () => {
+  let props;
+
+  beforeEach(() => {
+    props = {
+      isAttendantsLoading: false,
+      isJobTitlesLoading: false,
+      submitAttendant: jest.fn(),
+      jobTitles: jobTitles,
```

Comment:
job titles can be defined inline here. In general - if the variable is used once, it is worth considering defining it inline